KubeArmor/enforcer/bpflsm/enforcer.go

```diff
@@ -389,15 +403,7 @@ func (be *BPFEnforcer) DestroyBPFEnforcer() error {
 		errBPFCleanUp = true
 	}
 
-	for _, link := range be.Probes {
-		if link == nil {
-			continue
-		}
-		if err := link.Close(); err != nil {
-			be.Logger.Err(err.Error())
-			errBPFCleanUp = true
-		}
-	}
```

Comment:
Why are we doing this?

Reply:
Sorry, It was a change I did during testing.